neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -315,6 +310,42 @@ def ping_experiment(self, experiment):
     def create_hardware_metric(self, experiment, metric):
         pass
 
+    def log_artifact(self, experiment, artifact, destination=None):
+        target_name = os.path.basename(artifact) if destination is None else destination
+        target_name = f'{alpha_consts.ARTIFACT_ATTRIBUTE_SPACE}{target_name}'
+        dest_path = alpha_path_utils.parse_path(normalize_file_name(target_name))
+        if isinstance(artifact, str):
+            if os.path.exists(artifact):
+                operations = [alpha_operation.UploadFile(
+                    path=dest_path,
+                    file_name=dest_path[-1],
+                    file_path=os.path.abspath(artifact),
+                )]
+            else:
+                raise FileNotFound(artifact)
+        elif hasattr(artifact, 'read'):
+            if destination is not None:
+                operations = [alpha_operation.UploadFileContent(
+                    path=dest_path,
+                    file_name=dest_path[-1],
+                    file_content=base64_encode(artifact.read().encode('utf-8')),
+                )]
+            else:
+                raise ValueError("destination is required for file streams")
+        else:
+            raise ValueError("artifact is a local path or an IO object")
```

Comment:
It's copy pasted from old client, but I think that it should be "Artifact must be a local path or an IO object".
Can you confirm?